model/job.js

```diff
@@ -12,9 +9,15 @@ const MODEL = {
         .example('50dc14f719cdc2c9cb1fb0e49dd2acc4cf6189a0'),
 
     name: Joi
-        .string()
+        .string().regex(/^[A-Za-z0-9_-]+$/).min(3)
```

Comment:
If I'm a lazy user and want to create some quick job to test out screwdriver and use "dp" instead of deployment, this might annoy me. I think it's kinda unnecessary but I'm not super against it.